src/clientversion.h

```diff
@@ -38,7 +38,7 @@
 #define DO_STRINGIZE(X) #X
 
 //! Copyright string used in Windows .rc files
-#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The " PACKAGE_NAME " Developers"
```

Comment:
After thinking about this in detail, I kind of agree with @luke-jr: this line does not change the copyright of the sources itself. It's just a information string that will be transported to the user over package informations and it's under the responsibility of the package-/project-mainteiner(s).

If this (string) needs to fit all legal constraints, then it would probably require to add "LevelDB" and other third-party copyrights here.